README.md

```
@@ -1,192 +1,263 @@
+| 🎯 Quick Start Guide | [Get started in minutes](https://docs.opentiny.design/tiny-robot/guide/quick-start) |
+| 🎨 Theme Configuration | [Customize themes](https://docs.opentiny.design/tiny-robot/guide/theme-config) |
+| 📝 Update Log | [Version history](https://docs.opentiny.design/tiny-robot/guide/update-log) |
+| 💡 Examples | [Complete app examples](https://docs.opentiny.design/tiny-robot/examples/assistant) |
+| 🤖 TinyRobot Skills | [AI-assisted development](https://gitcode.csdn.net/69bb9b7f54b52172bc628048.html) |
+
+[![Ask DeepWiki](https://deepwiki.com/badge.svg)](https://deepwiki.com/opentiny/tiny-robot)
 
 ## 🏗️ Project Structure
 
-```text
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)